src/sciline/graph.py

```diff
@@ -0,0 +1,70 @@
+# SPDX-License-Identifier: BSD-3-Clause
+# Copyright (c) 2023 Scipp contributors (https://github.com/scipp)
+from typing import (
+    Any,
+    Callable,
+    Dict,
+    List,
+    NewType,
+    Tuple,
+    TypeVar,
+    get_args,
+    get_origin,
+)
+
+from graphviz import Digraph
+
+from .pipeline import Graph
+
+
+def to_graphviz(graph: Graph) -> Digraph:
```

Comment:
I think it would be a good idea to also add a test for this.

In addition, do we want to make it explicit that it is using `graphviz` (as it is currently), or is that an implementation detail that the user should not care about? Meaning we would call it something like `show_graph`, as we have done for the `transform_coords` and `plopp`.

Reply:
Sure, the main point of this PR is the other refactor.